CHANGELOG

```diff
@@ -14,14 +24,6 @@
   module.  See the selinux module to see how to use puppet to enable/disable
   selinux.  This may change the defaults for selinux in the `simp_lite`
   scenario.
-  
```

Comment:
rebase error

Reply:
@jeefberkey This is fine. Commits from a single person should be rolled up to the latest date in a given release.